Makefile

```diff
@@ -42,6 +42,15 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+build-contained:
+	if [ -f mender-artifact ]; \
+	then \
+		rm mender-artifact; \
+	fi; \
```

Comment:
You can simplify this even more by just running `rm -f mender-artifact`.

IMO, the least conditionals and for loops in Makefiles, the better. Only the strictly needed ones.

Reply:
my bad, I just thought of either regular `rm` (which potentially results in a warning) or reusing the condition... is `rm -f` now